instrument_config/obsolete_txt/pyuvdata_calfits_from_txt.py

```diff
@@ -0,0 +1,148 @@
+# An example script to generate calfits from pointing/cable calibration txt files,
+# modified from the example script on the pyuvdata github. 
+
+from pyuvdata import UVCal
+import numpy as np
+
+# Time array is in JD and calculated at the center of the time sample, and the corresponding pointing number.
+# Best to reference pointing JDs from Aug 23, 2013 (though not required)
+time_array = [2456528.21023,2456528.23208,2456528.25329,2456528.27444,2456528.29565]
+poi_array = ['-2','-1','0','1','2']
+#
+# large reference time array = [2456528.18694,2456528.21023,2456528.23208,2456528.25329,2456528.27444,2456528.29565,2456528.31894,2456528.34194]
+# for pointings ['-3','-2','-1','0','1','2','3','4']
+#
+Ntimes = len(time_array)
+
+# Frequency array of the cal in Hz
+freq_array = np.linspace(1.67075e8, 1.97715e8, 384)  # highband
+# freq_array for lowband is np.linspace(1.38915e8, 1.69555e8, 384)
+
+Nfreqs = len(freq_array)
+jones_array = np.array([-5, -6])  #  only 2 jones parameters, jxx and jyy.
+Njones = len(jones_array)
+ant_array = np.arange(128)
+Nants_data = len(ant_array)
+antenna_names = np.array(['ant{0}.format(ant)' for ant in ant_array])
+Nspws = 1  # only 1 spw is supported
+
+# Generate data arrays
+gains = np.zeros((Nants_data,Nfreqs,Ntimes,Njones)) + 0j
+flags = np.ones_like(gains, dtype=np.bool)
+chisq = np.ones_like(gains, dtype=np.float32)
+
+### Get cable lengths to sort calibration solutions to the right tiles
+datafile_name='/nfs/eor-00/h1/nbarry/MWA/IDL_code/FHD/instrument_config/mwa_cable_length.txt'
+datafile = open(datafile_name, "r")
+temp = next(datafile)
+cols = (col.strip().split() for col in datafile)
+rows = zip(*cols)
+
+tile_index = np.asarray(rows[0], dtype=np.int16)
+cable_length = np.asarray(rows[2], dtype=np.float32)
+
+datafile.close()
+
+index90 = np.where(cable_length == 90)
+index150 = np.where(cable_length == 150)
+index230 = np.where(cable_length == 230)
+index320 = np.where(cable_length == 320)
+index400 = np.where(cable_length == 400)
+index524 = np.where(cable_length == 524)
+###
+
+for poi_i in range(len(poi_array)):
+	poi = poi_array[poi_i]
```

Comment:
Not terribly important, but this could be written more succinctly as:
```
for poi_i, poi in enumerate(poi_arrary):
```

Reply:
done